syntax = "proto3";

service UserService {
  rpc GetUser(UserId)returns(User){}
}



message UserId {
  int32 Id = 1;
}

message User {
  string Name = 1;
}